protocol/js/flow-types-chat.js

```diff
@@ -169,6 +169,12 @@ export const NotifyChatChatActivityType = {
   failedMessage: 5,
 }
 
+export const RemoteMessageBoxedVersion = {
+  vnone: 0,
```

Comment:
Since the version field is new, existing messages will decode as `MessageBoxed.Version=0`. I was thinking that there will be values for version
- `VNONE=0` are V1 messages sent before https://github.com/keybase/client/pull/5965.
- `V1=1` are V1 messages sent after the addition of the version field (roughly)
- `V2=2` are V2 messages